kernel/common/binding.rb

```diff
@@ -54,4 +54,12 @@ def eval(expr, filename=nil, lineno=nil)
 
     Kernel.eval(expr, self, filename, lineno)
   end
+
+  def local_variables
+    eval('local_variables')
```

Comment:
Euh, we shouldn't need (nor want) the use of `eval` here. If I'm not mistaken the code currently used for `Kernel.local_variables` should be usable in this context, either in the form of a copy-paste (meh) or by using a helper method (one that takes a `Rubinius::VariableScope`).